utils/localization.js

```diff
@@ -142,7 +143,7 @@ padding: 6px 12px;
 margin: 0px;
 display: inline-block;
 margin-top: 8px;margin-bottom: 40px"
-href=${process.env.ADMIN_BASE_URI}confirmation/${token}>Перейти за посиланням</a>
+href=${ADMIN_BASE_URI}confirmation/${token}>Перейти за посиланням</a>
```

Comment:
It seems you forgot to import `ADMIN_BASE_URI`

Reply:
Solved.